Add support for redirectUrl on /create route

Patrick1904 opened the pull request:
**Changes:**
1. Add support for optional `redirectUrl`
2. Add `accountId` as url param to `redirectUrl`

**Example:**
`https://wallet.near.org/create?redirectUrl=https://berryclub.io/`

After account creation --> `https://berryclub.io/?accountId=johndoe.near`

**W/ linkdrop example:**
`https://wallet.near.org/create/near/VGmU9e2cEgE8PHysrBdjeJkbzvvFtBYT1UgEmDHTFTp81gnHSdKZCVyVpV5PwjYT6FX7Znd731HCvSpVnJJmgnx?redirectUrl=https://berryclub.io/`

After account creation --> `https://berryclub.io/?accountId=johndoe.near`

vgrichina commented:
nit: I'd prefer we call it `redirectUrl` as described here https://github.com/near/near-wallet/issues/1641 to avoid confusion with login `success_url` which is supposed to be deprecated when we implement https://github.com/near/near-wallet/issues/726

vgrichina commented on src/actions/account.js:
```diff
@@ -73,24 +73,23 @@ export const parseTransactionsToSign = createAction('PARSE_TRANSACTIONS_TO_SIGN'
         }
-
-        if ([WALLET_LOGIN_URL, WALLET_SIGN_URL].includes(currentPage) && search !== '') {
+        if (['create'].includes(currentPage) && search !== '') {
```
nit: let's use constant for 'create' like for other paths

vgrichina commented on src/actions/account.js:
```diff
@@ -414,18 +413,22 @@ export const handleCreateAccountWithSeedPhrase = (accountId, recoveryKeyPair, fu
         dispatch(redirectTo('/enable-two-factor', { globalAlertPreventClear: true }))
     } else {
-        dispatch(redirectToApp('/profile'))
+        if (url?.redirect_url.includes('/create') && url?.success_url) {
```
not sure how `redirect_url` comes into play here?

Patrick1904 commented:
> nit: I'd prefer we call it `redirectUrl` as described here #1641 to avoid confusion with login `success_url` which is supposed to be deprecated when we implement #726

Thanks @vgrichina. I simplified it a bit and fixed the naming. Please check it.
